src/main/webapp/WEB-INF/templates/fragments/head.html

```diff
@@ -8,13 +8,29 @@
 
     <script src="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/js/bootstrap.bundle.min.js" integrity="sha384-b5kHyXgcpbZJO/tY9Ul7kGkf1S0CWuKcCD38l8YkeH8z8QjE0GmW1gYU5S9FOnJ0" crossorigin="anonymous"></script>
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.5.1/jquery.min.js"></script>
-    <script src="static/js/collector.js" type="text/javascript"></script>
-    <script type="text/javascript">
-        var $SCRIPT_ROOT = "http://localhost:8080/SecureDataCollector-1.0-SNAPSHOT";
-    </script>
 
-    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.0-beta2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-BmbxuPwQa2lc/FVzBcNJ7UAyJxM6wuqIj61tLrc4wSX0szH/Ev+nYRRuWlolflfl" crossorigin="anonymous">
-    <link href="static/css/style.css" rel="stylesheet" type="text/css">
+    <!-- KEDZE TU SA VYUZIVA THYMELEAF MUSI BYT TH: -->
+    <th:block th:if="${trueIfRunningRemotely == false}">
```

Comment:
Cele to vyhodnocovanie if logiky sa mi nepaci ze sa robi v template. Urob if v jave a sem uz iba posli hotove rozhodnutie, takto nejako:

```
<script src="${base_url}/static/js/collector.js" type="text/javascript"></script>
<script type="text/javascript">
    var $SCRIPT_ROOT = "${base_url}";
</script>

<link href="${base_url}/static/css/style.css" rel="stylesheet" type="text/css">
<link rel="icon" href="${base_url}/static/images/favicon-32.png" type="image/x-icon">
```

Reply:
spravene